lib/canvas_exporting/exporter.rb

```diff
@@ -76,7 +90,7 @@ def convert_args(args)
     end
 
     def convert_js_path
-      File.join(ROOT, 'phantomjs/highcharts-convert.js').to_s
+      File.join(ROOT, 'phantomjs/canvas-convert.js').to_s
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.